tests/schema/test.firefox-schemas-import.js

```diff
@@ -1152,15 +1146,12 @@ describe('firefox schema import', () => {
         .stub(tar, 'Parse')
         .returns(extractedStream);
       expect(fs.readdirSync(outputPath)).toEqual([]);
-      return fetchSchemas({ inputPath: 'mozilla-central.tgz', outputPath })
-        .then(() => {
-          expect(true).toBeFalsy();
-        }, () => {
-          expect(true).toBeTruthy();
-        });
+      await expect(
+        fetchSchemas({ inputPath: 'mozilla-central.tgz', outputPath })
+      ).rejects.toThrow();
```

Comment:
I'm wondering if we could also assert something about what error is expected to be raised here.

@EnTeQuAk what do you think? is it worth it to make this assertion more detailed and also verify the expected error message?